versions.yaml

```diff
@@ -40,7 +40,15 @@ git:
   opa:
     url: https://github.com/open-policy-agent/opa
     reference: v0.58.0
+  kbs:
+    url: https://github.com/confidential-containers/kbs
+    reference: 7ec54fa8f74b9bf24d84ef4a35d050d068105732
 oci:
   pause:
     registry: docker://registry.k8s.io/pause
     tag: 3.6
+
+image:
+  kbs:
+    registry: ghcr.io/confidential-containers/key-broker-service
+    tag: 7ec54fa8f74b9bf24d84ef4a35d050d068105732
```

Comment:
This tag probably does not exists:

`coco-tenant                      0s          Warning   Failed                        pod/kbs-5864d9656d-r25nk                                    Failed to pull image "ghcr.io/confidential-containers/key-broker-service:7ec54fa8f74b9bf24d84ef4a35d050d068105732": rpc error: code = NotFound desc = failed to pull and unpack image "ghcr.io/confidential-containers/key-broker-service:7ec54fa8f74b9bf24d84ef4a35d050d068105732": failed to resolve reference "ghcr.io/confidential-containers/key-broker-service:7ec54fa8f74b9bf24d84ef4a35d050d068105732": ghcr.io/confidential-containers/key-broker-service:7ec54fa8f74b9bf24d84ef4a35d050d068105732: not found`

Reply:
The correct registry is `ghcr.io/confidential-containers/staged-images/kbs`.